warehouse/README.md

```diff
@@ -200,37 +197,72 @@ If you prefer to install dbt locally and use your own development environment, y
 ### Install the Google SDK (if you haven't)
 
 0. Implied: make sure that you have GCP permissions (i.e. that someone has added you to the GCP project)
-1. Follow the installation instructions at [https://cloud.google.com/sdk/docs/install](https://cloud.google.com/sdk/docs/install)
-   1. If this is your first time using the terminal, we recommend reading [https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line](https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line)
-      or another tutorial first. You will generally need to understand `cd` and
-      the concept of the "home directory" aka `~`. When you first open the
-      terminal, your "working directory" will be the home directory. Running the
-      `cd` command without any arguments will set your working directory back to
-      `~`.
-   2. Use `tar -xvf <drag file from Finder window to get file name> ~` to unzip into your home directory
-   3. Answer `yes` to adding the tool to your path
-   4. Most recent macOS versions use `zsh` as the default shell, so ensure the path modification is added to `~/.zshrc` when prompted
-   5. Restart your terminal, and run `gcloud init`
-   6. Answer `yes` to log in, and select the Google account associated with GCP
-   7. Set `cal-itp-data-infra` as the default project, and do not set a region
-2. You should also [set the application default](https://cloud.google.com/sdk/gcloud/reference/auth/application-default)
-3. If `bq ls` shows output, you are good to go.
+
+1. Install and configure Google Cloud CLI
+
+   1. Follow the installation instructions in [Google `gcloud` documentation](https://cloud.google.com/sdk/docs/install)
```

Comment:
I think it might be helpful to leave the more explicit notes about how to unzip and add it to your `PATH`? Since that's not spelled out as clearly in the linked Google documentation

Reply:
Good call about adding back the unzipping! As for the `PATH` edits, it _sounds_ to me like Google's install script will modify your `PATH` for you?

![image](https://github.com/cal-itp/data-infra/assets/1246453/c06c21e4-0f3b-4bc4-8880-7bb5a6e9a127)

Reply:
Ah my apologies, I re-read what was there originally and misunderstood what it was describing. I've added it back in add58356029dbc1c963b0ee924a410d8071aeab2 😄